.github/workflows/contracts.yml

```diff
@@ -81,4 +82,69 @@ jobs:
         run: npm ci
 
       - name: Lint
-        run: npm run lint
\ No newline at end of file
+        run: npm run lint
+
+  migrate-and-publish-celo:
+    needs: [build-and-test, lint]
+    # FIXME: alfajores branch and pull request events are for test purposes only - revert once done
+    if: |
+      (startsWith(github.head_ref, 'releases/alfajores/')
+        || github.ref == 'refs/heads/master')
+        && (github.event_name == 'push'
+        || github.event_name == 'pull_request'
+        || github.event_name == 'workflow_dispatch')
```

Comment:
Can the final config be applied now? Or do we want to remove it in different PR?

Reply:
Please see https://github.com/keep-network/tbtc/commit/41be3be7830fef652f28e27835ebc0803443fb21